app/src/main/kotlin/org/fossify/phone/fragments/FavoritesFragment.kt

```diff
@@ -183,9 +183,15 @@ class FavoritesFragment(context: Context, attributeSet: AttributeSet) : MyViewPa
             val orderIds = items.map { it.contactId }
             val orderGsonString = Gson().toJson(orderIds)
             config.favoritesContactsOrder = orderGsonString
+            assignCustomOrderToAllContact(items)
         }
     }
 
+    private fun assignCustomOrderToAllContact(orderIds: List<Contact>){
+        allContacts.clear()
+        allContacts.addAll(orderIds)
+    }
+
```

Comment:
Instead of clearing and adding to the original list, create a copy:
```suggestion
            allContacts = ArrayList(items)
        }
    }

```

You don't need `assignCustomOrderToAllContact()` method for it.